cmd/dispatcher/main.go

```diff
@@ -74,6 +75,11 @@ func main() {
 		logger.Fatalf("Failed to build coordination client: %s", err.Error())
 	}
 
+	redisPW, err := ioutil.ReadFile("redis.conf")
+	if err != nil { //wasnt able to find the path to the file/not sure where it is
+		logger.Fatalf("Failed to retrieve database password from a file: %s", err.Error())
+	}
+
```

Comment:
Since you have now the password here, please initialize the redis client in this point, and pass the whole client to the operator.

Reply:
https://github.com/vega-project/ccb-operator/blob/master/pkg/dispatcher/operator/operator.go#L62-L66
Are you talking about this one? Because I am not really sure about it, when i put in main.go, then the functions underneath are saying that redisClient is undefined.

Reply:
Also I dont know which operator do you mean. Like the Operator struct or ?

Reply:
I mean to pass it to `operator.NewMainOperator`.

Reply:
Then the redisClient can be a part of the Operator struct. Then you can replace all of its references